shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/dialect/statement/oracle/ddl/OracleAlterIndexTypeStatement.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.ddl;
+
+import org.apache.shardingsphere.sql.parser.sql.common.statement.AbstractSQLStatement;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.oracle.OracleStatement;
+
+/**
+ * Oracle alter index type statement.
+ */
+
+public final class OracleAlterIndexTypeStatement extends AbstractSQLStatement implements OracleStatement {
```

Comment:
@chakkk309 Please add @ToString annotation for OracleAlterIndexTypeStatement. When the user turns on the `sql-show` parameter, the object string should be printed, not the memory address.

Reply:
In addition, is it possible to add the index name in OracleAlterIndexTypeStatement? In sharding feature, ShardingSphere will rewrite the index name.

Reply:
> In addition, is it possible to add the index name in OracleAlterIndexTypeStatement? In sharding feature, ShardingSphere will rewrite the index name.

Do you mean the `index type name`?

Reply:
@chakkk309 What is the meaning of index type?